packages/react-api/src/overrides/spec/edgeware.ts

```diff
@@ -2,11 +2,10 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
```

Comment:
Makes more sense to remove the edgeware stuff altogether

Reply:
sure.. will remove Edgware completely.. anyways its doesn't help us